packages/shared_preferences/shared_preferences_platform_interface/lib/shared_preferences_platform_interface.dart

```diff
@@ -62,11 +62,21 @@ abstract class SharedPreferencesStorePlatform extends PlatformInterface {
   /// * Value type "StringList" must be passed if the value is of type `List<String>`.
   Future<bool> setValue(String valueType, String key, Object value);
 
-  /// Removes all keys and values in the store.
+  /// Removes all keys and values in the store with default prefix.
```

Comment:
This assumes the reader knows what the default prefix is, and why there is one. Someone writing a new implementation may well not, so we should elaborate. E.g.

```
/// Removes all keys and values in the store where the key starts with 'flutter.'.
///
/// This default behavior is for backwards compatibility with older versions of this
/// plugin, which did not support custom prefixes, and instead always used the
/// prefix 'flutter.'.